### DISCUSSION TITLE
Uploading CSV files and sending to Express backend

### DISCUSSION THREAD

**Discussion by adamkudz** [quasarframework/quasar#15043] [General - Components / Directives / etc]
Hello,

I'm trying, without success, to upload multiple csv files to my backend for processing with Multer and Parse-csv. The backend works great when I submit the form through standard HTML, but I am unable to get it working with Quasar.  Do I need to create a new formData object and append?

Any help would be greatly appreciated.  Thanks.

Adam

**Comment by MilosPaunovic** [comment]
Could you share what have you already tried?

**Reply by adamkudz** [reply]
I ended up using the Q-Uploader.  The deep dive into the world of Forms was instructional, and I'm grateful for that. But...I'd prefer to keep building at this point.  :)

**Comment by suleimanas** [comment]
yes you already have the answer `new formData`
